build/linux/Makefile

```diff
@@ -87,6 +87,12 @@ INSTALLER_TMPDIR := $(INTERMEDIATE_DIR)/installer_tmp
 
 # GO Source dir for custom fluent bit plugin
 GO_SOURCE_DIR := $(SOURCE_DIR)/plugins/go/src
+GO_SOURCE_INPUT_DIR := $(SOURCE_DIR)/plugins/go/input
+
+CMETRICS_VERSION = 0.6.0
+AMD64_DEB_HEADERS = cmetrics_$(CMETRICS_VERSION)_amd64-headers.deb
+AMD64_DEB = cmetrics_$(CMETRICS_VERSION)_amd64.deb
+AMD64_URL = https://github.com/fluent/cmetrics/releases/download/v$(CMETRICS_VERSION)
```

Comment:
do we have these dependencies for windows? cc: @jatakiajanvi12

Reply:
resolving since Janvi built this seperately for windows